src/data/markdown/translated-guides/en/02 Using k6/06 Test life cycle.md

```diff
@@ -26,10 +31,26 @@ export function teardown(data) {
 
 </CodeGroup>
 
+## A quick overview of test stages
+
+This table provides the essential information about each stage.
+The rest of this page goes into deeper technical detail.
+
+| Test stage      | Used to                                                    | Example                                                                                 | Called                                                                             | Required? |
+|-----------------|------------------------------------------------------------|-----------------------------------------------------------------------------------------|------------------------------------------------------------------------------------|-----------|
+| **1. init**     | Load local files, import modules, declare global variables | Open JSON file, Import module                                                           | Once per VU\*                                                                      | Optional  |
```

Comment:
Init is optional, but a test without importing any k6 API is useless.  Should Init be required ??

Reply:
Init isn't optional. You always need init code as this is the place that defines which function is the default ;).
All the code that is outside of a function will be executed *always* at the begining, including if a function is called

```javascript
function one() {
  // this code is not init context and isn't called so it won't be executed 
}
function two() {
  // this code is not the init context but is called
}

two() // this is in the init context and calles the function `two` so it's code will also be executed

// export default function() {} // if this isn't uncommented k6 will give you error because nothing was defined as the function to be executed 

export function setup() { // this line that is in the "init code" is what defines that there is `setup` to be called
  // some setup code
} 
```
I hope this makes it clear that "init code" is totally not optional ;). 

This actually is one of the reason (IMO) that we don't call it init code, but init context. As that moves the thinking away from code and to *where* the code is being executed (in the init context). 

I would expect there is probably even a better way to explain this using words that are more specific and don't let so much for interpretation. 

WDYT @na--